### PR TITLE
Update UG and storage

### PR DESCRIPTION
**Pull request by hshiah**
Update UG. Fix some minor errors of reminder and goal. Update the saving and loading for goals and reminder.

### CODE REVIEW

File: src/main/java/seedu/financialplanner/goal/Goal.java [hunk 1]
```diff
@@ -27,7 +37,7 @@ public int getAmount() {
         return this.amount;
     }
     public String formatString() {
-        String status = isDone ? "Done" : "Not Done";
-        return this.label + " | " + this.amount + " | " + this.isDone;
+        String status = isDone ? "Achieved" : "Not Achieved";
```
Comment (NeoMinWei) [inline src/main/java/seedu/financialplanner/goal/Goal.java:40]: Nice use of ? and :

File: src/main/java/seedu/financialplanner/storage/LoadData.java [hunk 2]
```diff
@@ -61,6 +67,14 @@ public static void load(String filePath, LocalDate date) throws FinancialPlanner
...
                     loadBudget(split);
                     break;
+                case "R":
+                    final Reminder reminder = getReminder(split);
+                    reminderList.load(reminder);
+                    break;
+                case "G":
+                    final Goal goal = getGoal(split);
+                    wishList.load(goal);
+                    break;
```
Comment (NeoMinWei) [inline src/main/java/seedu/financialplanner/storage/LoadData.java:77]: logging can be added here when default case is reached. level severe

**Review by NeoMinWei — APPROVED** [review]
LGTM

File: src/main/java/seedu/financialplanner/goal/WishList.java [hunk 3]
```diff
@@ -28,7 +28,12 @@ public void deleteGoal(int index) {
     public String toString() {
         String result = "";
         for (int i = 0; i < list.size(); i++) {
-            result += String.format("%d. %s\n", i + 1, list.get(i));
+            if (i == list.size() - 1) {
+                result += String.format("%d. %s", i + 1, list.get(i));
+                break;
```
Comment (ryan1604) [inline src/main/java/seedu/financialplanner/goal/WishList.java]: Redundant break statement?

File: src/main/java/seedu/financialplanner/reminder/ReminderList.java [hunk 4]
```diff
@@ -27,7 +27,12 @@ public void deleteReminder(int index) {
     public String toString() {
         String result = "";
         for (int i = 0; i < list.size(); i++) {
-            result += String.format("%d. %s\n", i + 1, list.get(i));
+            if (i == list.size() - 1) {
+                result += String.format("%d. %s", i + 1, list.get(i));
+                break;
```
Comment (ryan1604) [inline src/main/java/seedu/financialplanner/reminder/ReminderList.java]: also here

**Review by ryan1604 — APPROVED** [review]
LGTM

**Pull request merged**